### PR TITLE
Added folders with few scripts to begin with

### PR DESCRIPTION
**Pull request by NaseemSrour**
Added folders with initial scripts.

### CODE REVIEW

**Comment by NaseemSrour** [comment]
Left to-do (might create as a new issue):
1. Test on another machine (after manually installing MySQL on that machine and creating a MySQL username). 
2. Create the dummy data scripts.

**Review by rudyeila — APPROVED** [review]
Looks good. 

Moving forward, I'd suggest adding a class representation of each DB object into the models folder. That way, we can create a python object and pass that to the DB controller, which in turn stores it into the DB, and vice versa. When retrieving an object from the DB, you'd already have a python class representation of it that can be used.

File: server/database/config.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import sys
+
+db_username = 'root'
+db_password = 'sqlninja678'
+db_host = 'localhost'
+db_name = 'delivery'
+
+sys.path.append(".")
```
Comment (rudyeila) [inline server/database/config.py:8]: I like the usage of a config file, it is more practical and better from a security point of you. 

I think it is possible to keep this line out, if you make the database folder a module by adding an empty __init__.py file. Then you can either do relative imports from other folders inside the database, or something like from database.config import db_username etc...

File: server/database/db_controller.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import mysql.connector
+from mysql.connector import errorcode
+import json
+import config
```
Comment (rudyeila) [inline server/database/db_controller.py:4]: This is what I was referring to with the other comment... If you make database a module then I think here you can do: 
import database.config and you can keep out the sys.path.append line. I think at least...

**Comment by rudyeila** [comment]
@NaseemSrour Also, please remember to add reviewers to the PR, so that we get notified.

File: server/database/config.py (hunk 1 above)
Comment (NaseemSrour) [reply server/database/config.py:8]: I'll search for how to make it a module, in order to make the imports inside the database more intuitive like you stated.

**Comment by NaseemSrour** [comment]
Good notice on the class representation of the DB objects!
Do you think we should define it as a Task? Since it requires an amount of work and preferably not get done under the name of this PR.

Yes @majdya notified me about assigning you as reviewer on future PRs, I thought it does that automatically since you both are already project contributors. No problem.

**Review by majdya — APPROVED** [review]
good progress, I know this is an init phase thats why we have a lot of files changed at once, but for further PR lets keep the number at minimum, this will ease the review process.

@rudyeila @NaseemSrour

File: server/database/config.py (hunk 1 above)
Comment (rudyeila) [reply server/database/config.py:8]: Sure, you can also create a task for that and then it can be done separately.

**Comment by rudyeila** [comment]
> 
> 
> Good notice on the class representation of the DB objects!
> Do you think we should define it as a Task? Since it requires an amount of work and preferably not get done under the name of this PR.
> 
> Yes @majdya notified me about assigning you as reviewer on future PRs, I thought it does that automatically since you both are already project contributors. No problem.
> 
> Majd do not merge this PR yet, I want to solve the Module issue tomorrow or the day after. I'll commit, we'll review, and then merge it.

Yes that should be done as another task.

File: server/database/config.py (hunk 1 above)
Comment (NaseemSrour) [reply server/database/config.py:8]: it needs more discussion, so I'll create it as a task.

**Pull request merged**